Explorer/Assets/DCL/WebRequests/Texture/ArgsFactory/GetTextureArgsFactory.cs

```diff
@@ -0,0 +1,22 @@
+using Plugins.TexturesFuse.TexturesServerWrap.Unzips;
+
+namespace DCL.WebRequests.ArgsFactory
+{
+    public class GetTextureArgsFactory : IGetTextureArgsFactory
```

Comment:
I would propose you inject `ITexturesUnzip` on the level of the web controller as this is not a variable argument for every texture.

E.g. you can move
```
internal static readonly InitializeRequest<GetTextureArguments, GetTextureWebRequest> GET_TEXTURE = GetTextureWebRequest.Initialize;
```

Into `IWebRequestController` making it non-static and inject this dependency there (as a capture in a delegate).

It will be cleaner then injecting `GetTextureArgsFactory` everywhere in order to bypass this limitation.